frontend/index.js

```diff
@@ -1,5 +1,24 @@
-const dateHeader = document.getElementById('dateHeading');
+const addTodaysDate = () => {
+    const dateToday = document.createTextNode(String((new Date()).toLocaleString()));
+    return dateHeader = document.getElementById('dateHeading')
+                                .append(dateToday);
+};
 
-const dateToday = document.createTextNode(String(new Date()));
+const addTimeZoneDifference = () => {
+    const usersHoursFromGMT = ((new Date()).getTimezoneOffset())/60;
+    const usersHoursFromKennedy = usersHoursFromGMT + 5;
 
-dateHeader.append(dateToday);
\ No newline at end of file
+    const kennedyTimeHeader = document.getElementById('kennedyTime');
+    const kennedyTimeZoneOffset = document.createTextNode(usersHoursFromKennedy);
+    kennedyTime.append(kennedyTimeZoneOffset);
+
+    const aheadBehindText = usersHoursFromKennedy > 0 ? "ahead of" : "behind"
+
+    const hoursAheadBehind = document.createTextNode(` hours ${aheadBehindText} the Kennedy Space Centre`);
+
+    return kennedyTimeHeader.append(hoursAheadBehind);
```

Comment:
Could also chain together this `append` with the `getElementById`

Also, should this return anything?

Reply:
I've chained this together and removed the returns in both functions. :)